templates/errors/401.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'layouts/_base.html' %}
+
+{% set page_title = _("Page is not available") %}
+
+{% block main %}
+  <h1>{{ _("Sorry, you need to sign in again") }}</h1>
+  <p>{{ _("This is because you have either:") }}</p>
+  <ul>
+    <li>{{ _("been inactive for 45 minutes and your session has timed out to protect your information") }}</li>
+    <li>{{ _("followed a link to a page you are not signed in to") }}</li>
```

Comment:
On the overflow, this is `followed a link to a serivce you are not signed in to`, instead of `page`.

Reply:
Overflow text is not correct , It was updated from `service` to `page`,  see the card [comments](https://trello.com/c/wJmbDMQo/4909-introduce-401-unauthorised-page-2-sp#comment-61658c832ff96f1c9d53a9fe) by Sarah